packages/nextjs/src/copy-config-files-exec.ts

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env node
```

Comment:
Had to remove because rollup was complaining. It's added after rollup is done (see comment above).